pkg/resources/external_function.go

```diff
@@ -89,6 +85,7 @@ var externalFunctionSchema = map[string]*schema.Schema{
 		Optional:    true,
 		ForceNew:    true,
 		Description: "Indicates whether the function can return NULL values or must return only NON-NULL values.",
+		Default:     true,
```

Comment:
Why? It seems like a breaking change. If it was missing earlier, `returnNullAllowed` was not set in the `ExternalFunctionBuilder`, so it was false by default and the resulting query had `NOT NULL`:
```
	if !fb.returnNullAllowed {
		q.WriteString(` NOT`)
	}
	q.WriteString(` NULL`)
```

Reply:
> "Indicates whether the function can return NULL values or must return only NON-NULL values"
if true then it can return null value
if false it can only return non-null values.
In documentation
>> Default: The default is NULL (i.e. the function can return NULL values).
So if you would like me to make this false to keep with the old behavior, I can or else I have updated the MIGRATION DOCS with the change

Reply:
I don't like introducing so many behavior changes in the PR which was supposed to just use the new SDK implementation, and also knowing, that we will rework every resource in the upcoming months. This means potentially having behavioral changes multiple times.

Let's leave this as a behavior change this time. Close after reading.